crux-bench/src/crux/bench/watdiv_datomic.clj

```diff
@@ -300,17 +300,17 @@
          (vec))))
 
 (defn with-datomic [f]
-  (let [uri (str "datomic:free://"
-                 (or (System/getenv "DATOMIC_TRANSACTOR_URI") "datomic")
-                 ":4334/bench?password=password")]
+  (let [uri (str "datomic:mem://bench")]
     (try
         (d/delete-database uri)
         (d/create-database uri)
-        (with-open [conn ^Closeable (d/connect uri)]
+        (let [conn (d/connect uri)]
           @(d/transact conn datomic-watdiv-schema)
-          (f conn))
+          (f conn)
+          (d/release conn))
```

Comment:
should `release` be in the `finally` block?

Reply:
Potentially, though would need to move the *let*